designsafe/static/scripts/data-depot/components/community/community.component.js

```diff
@@ -0,0 +1,126 @@
+import _ from 'underscore';
+import CommunityTemplate from './community.component.html';
+
+export function CommunityDataCtrl($scope, $state, Django, DataBrowserService) {
```

Comment:
Add
```
'ngInject';
```

Reply:
I've added all the `ngInject` annotations and made sure it works with `npm run build`. 

Can you clarify what you mean by "add comments on the controllers to deprecate the use of $scope" ? Those controllers still use $scope (that will hopefully change soon though), I only replaced $scope variables with service variables when state was being shared between components.

Reply:
I mean to add an explicit comment that we need to do that. But i think by now we all know we have to do it. So, it's ok :)